ex03.ex

```diff
@@ -101,7 +110,14 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([],[]), do: true	#The 2 lists are empty lists then they are equal.
```

Comment:
## -2

Good analysis of the different cases. However, you could use pattern matching to make the code simpler:

~~~ elixir
  def list_equal([],[]),         do: true
  def list_equal([h|t1],[h|t2]), do:  list_equal(t1, t2)
  def list_equal([_|_],[_|_]),   do: false
  def list_equal([],[_]),        do: false
  def list_equal([_],[]),        do: false
~~~